hyperdome/common/common.py

```diff
@@ -107,20 +107,36 @@ def tor_paths() -> typing.Tuple[Path, Path, Path, Path]:
     )
 
 
+MAX_PORT = 65535
+MIN_PORT = 0
+MAX_PORT_RETRY = 100
+
+
 @autologging.traced
 @autologging.logged
 def get_available_port(min_port: int, max_port: int) -> int:
     """
     Find a random available port within the given range.
     """
+    if not (isinstance(min_port, int) and isinstance(max_port, int)):
+        raise TypeError("ports must be integers")
+    if not (
+        MIN_PORT < min_port < MAX_PORT
+        and MIN_PORT < max_port < MAX_PORT
+        and min_port < max_port - 1
```

Comment:
Couldn't this be `MIN_PORT < min_port <= max_port < MAX_PORT`?

Reply:
I figured there was a better way, let me check if that still passes all tests.